parlai/agents/transformer/modules.py

```diff
@@ -99,9 +108,26 @@ def __init__(self, opt, dictionary):
         if n_positions < 0:
             raise ValueError('n_positions must be positive')
 
+        if opt.get('gelu_activation'):
```

Comment:
I would prefer if we had an `--activation` option with choices `{'relu', 'gelu'}`. That would make it easier for us to introduce another activation of choice later.